UpdateUser/handler.ts

```diff
@@ -4,8 +4,8 @@ import { User } from "@azure/graph/esm/models";
 import * as express from "express";
 import { toError } from "fp-ts/lib/Either";
 import { identity } from "fp-ts/lib/function";
-import { fromLeft } from "fp-ts/lib/TaskEither";
-import { taskEither, TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
+import { fromEither } from "fp-ts/lib/TaskEither";
+import { TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
```

Comment:
no need to import twice
```suggestion
import { fromEither, TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
```

Reply:
It' s so strange that lint doesn't report a non grouped import 😄